### ISSUE TITLE
cheyenne and DAV system updates cause PyNIO error

### ISSUE THREAD

**Issue by bertinia** [NCAR/CESM_postprocessing#209]
The system modules and ncar_pylib (v 20190627) updates on cheyenne cause the following PyNio error:

```
Traceback (most recent call last):
  File "./ocn_avg_generator.py", line 7, in <module>
    exec(compile(f.read(), __file__, 'exec'))
  File "/glade/work/aliceb/sandboxes/dev/postprocessing_new/diagnostics/diagnostics/ocn/ocn_avg_generator.py", line 43, in <module>
    from pyaverager import specification, PyAverager
  File "/glade/work/aliceb/sandboxes/dev/postprocessing_new/cesm-env2/lib/python2.7/site-packages/pyaverager/__init__.py", line 3, in <module>
    import PyAverager, specification, PreProc
  File "/glade/work/aliceb/sandboxes/dev/postprocessing_new/cesm-env2/lib/python2.7/site-packages/pyaverager/PreProc.py", line 1, in <module>
    import Nio
  File "/glade/work/aliceb/sandboxes/dev/postprocessing_new/cesm-env2/lib/python2.7/site-packages/PyNIO/Nio.py", line 83, in <module>
    from _nio import *
ImportError: libjpeg.so.62: cannot open shared object file: No such file or directory
MPT ERROR: could not run executable.
	(HPE MPT 2.19  02/23/19 05:31:12)
```

**Comment by bertinia** [comment]
CISL made a change to the underlying ncarenv module links so this issue should be fixed. 

New issue related to MPT launch error is now a problem. 
From /glade/work/aliceb/sandboxes/runs/b.e21.BHIST.f09_g17.CMIP6-historical.011/atm_averages:

```
[aliceb@cheyenne4:logs]>cat atm_averages.log.20190708-105129 
MPT: Launcher network accept (MPI_LAUNCH_TIMEOUT) timed out
MPT: Launcher on r3i2n24 failed to receive connection(s) from: r3i2n24.ib0.cheyenne.ucar.edu r3i2n25.ib0.cheyenne.ucar.edu r3i2n26.ib0.cheyenne.ucar.edu r3i2n34.ib0.cheyenne.ucar.edu
MPT ERROR: could not launch executable
	(HPE MPT 2.19  02/23/19 05:31:12)

```

**Comment by bertinia** [comment]
This MPT launch error was because there were some partial dirs/files in place. The atm_avg_generator.py should have been able to detect this and exit gracefully.

**Comment by bertinia** [comment]
fixed in PR #210
